profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/OsSpecificApi.cpp

```diff
@@ -36,4 +38,133 @@ StackFramesCollectorBase* CreateNewStackFramesCollectorInstance(ICorProfilerInfo
     return new Windows32BitStackFramesCollector(pCorProfilerInfo);
 #endif
 }
+
+uint64_t GetThreadCpuTime(ManagedThreadInfo* pThreadInfo)
+{
+    uint64_t duration = 0;
```

Comment:
nitpick: this variable is not used. you can remove duration and `return 0` at line 56

Reply:
✅